components/blocks/Landing/LandingLeaderboards.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+import type { ApiResponse } from '~/composables/useApi'
+import type { LeaderboardViewModel } from '~/lib/api/data-contracts'
+import { useLocalePath } from '#imports'
+const localePath = useLocalePath()
+
+interface LandingLeaderboardsProps {
+  leaderboards: ApiResponse<LeaderboardViewModel[]>
```

Comment:
Using this ApiResponse class as the property type feels a bit weird, especially since the component only uses the data property. Consider simply taking a list of LBs as props instead.